src/main/resources/assets/alchemistry/lang/fr_fr.json

```diff
@@ -0,0 +1,66 @@
+{
+  "tooltip.alchemistry.evaporator": "Fonctionne %d plus lentement dans les biomes humides.",
+  "tooltip.alchemistry.energy_requirement":"Requiert %d FE/t.",
+
+  "block.alchemistry.chemical_dissolver": "Dissolveur chimique",
+  "block.alchemistry.chemical_combiner": "Mélangeur chimique",
+  "block.alchemistry.evaporator": "Évaporateur",
+  "block.alchemistry.atomizer": "Atomiseur",
+  "block.alchemistry.liquifier": "Liquéfacteur",
+  "block.alchemistry.fission_controller": "Contrôleur de chambre à fission",
+  "block.alchemistry.fission_core": "Cœur de chambre à fission",
+  "block.alchemistry.fission_casing": "Coffrage de chambre à fission",
+  "block.alchemistry.fusion_controller": "Contrôleur de chambre à fusion",
+  "block.alchemistry.fusion_core": "Cœur de chambre à fusion",
+  "block.alchemistry.fusion_casing": "Coffrage de chambre à fusion",
+
+  "item.alchemistry.mineral_salt": "Sel minéral",
+  "item.alchemistry.fertilizer": "Engrais",
+  "item.alchemistry.obsidian_breaker": "Brise-obsidienne",
+  "item.alchemistry.milk": "LAIT",
+  "item.alchemistry.condensed_milk": "Lait condensé",
+  "item.alchemistry.slot_filler": "Bouche-trou",
+
+  "alchemistry.jei.combiner": "Mélangeur chimique",
+  "alchemistry.jei.dissolver": "Dissolveur chimique",
+  "alchemistry.jei.electrolyzer": "Électrolyseur",
+  "alchemistry.jei.evaporator": "Évaporateur",
+  "alchemistry.jei.atomizer": "Atomiseur",
+  "alchemistry.jei.liquifier": "Liquéfacteur",
+  "alchemistry.jei.fission": "Chambre à fission (multiblock)",
+  "alchemistry.jei.fusion": "Chambre à fusion (multiblock)",
+
+  "itemGroup.alchemistry": "Alchemistry",
+
+  "item.alchemistry.fertilizer.tooltip": "Peut être utilisé comme de la poudre d'os.",
+  "item.alchemistry.obsidian_breaker.tooltip": "Cliquez droit sur un bloc d'obsidienne pour la récolter immédiatement. Cet objet est à usage unique.",
+  "item.alchemistry.milk.tooltip": "All I do is milk.",
```

Comment:
Tout ce que je fais est du lait

Reply:
This tooltip is meant to be an easter egg. Therefore there is no reason to translate it.